packages/@ember/-internals/views/lib/views/core_view.ts

```diff
@@ -42,7 +47,7 @@ const CoreView = FrameworkObject.extend(Evented, ActionHandler, {
   */
   parentView: null,
 
-  instrumentDetails(hash) {
+  instrumentDetails(hash: any) {
```

Comment:
Fine to do a follow-on pass here, but would `Record<string, unknown>` work as a first pass?